ote/src/clj/ote/services/users.clj

```diff
@@ -81,31 +83,32 @@
           ;; Password incorrect, username or email taken => return errors to form
           (or username-taken? email-taken? password-incorrect?)
           {:success? false
-           :username-taken (when username-taken? (:username form-data))
-           :email-taken (when email-taken? (:email form-data))
+           :username-taken (when username-taken? new-username)
+           :email-taken (when email-taken? new-email)
            :password-incorrect? password-incorrect?}
 
           ;; Request is valid, do update
-          (let [user (specql/update! db ::user/user
+          (let [_ (specql/update! db ::user/user
```

Comment:
Could the `update!` call be moved outside the let?

Reply:
Sure if it's what we want to do. This pattern has been used quite a bit though.